db/migrate/20240108184133_create_users.rb

```diff
@@ -5,7 +5,7 @@ def change
       t.string :photo
       t.text :bio
       t.string :posts_counter
```

Comment:
- [ ] You do not have a `schema.rb` file. You need to run the following command so that your migrations will be updated and the database schema created/updated.

  ```
  rails db:migrate
  ```